xmtp_networking/src/grpc_api_helper.rs

```diff
@@ -74,9 +74,9 @@ impl Client {
             let connector = get_tls_connector();
 
             let tls_conn = hyper::Client::builder()
-                .pool_idle_timeout(Duration::from_secs(30))
-                .http2_keep_alive_interval(Duration::from_secs(10))
-                .http2_keep_alive_timeout(Duration::from_secs(5))
+                .pool_idle_timeout(Duration::from_secs(5))
```

Comment:
Are we sure this value is related to the bug in RN? If we kill idle connections faster, that could result in a lot more connections to our server and worse performance. But I don't really know.